include/dlaf/util_tile.h

```diff
@@ -0,0 +1,26 @@
+//
+// Distributed Linear Algebra with Future (DLAF)
+//
+// Copyright (c) 2018-2019, ETH Zurich
+// All rights reserved.
+//
+// Please, refer to the LICENSE file in the root directory.
+// SPDX-License-Identifier: BSD-3-Clause
+//
+
+/// @file
+
+#pragma once
+
+#include "dlaf/common/buffer.h"
+#include "dlaf/tile.h"
+
+namespace dlaf {
+
+/// @brief Create a common::Buffer from a Tile
+template <class T, Device device>
+auto create_buffer(const dlaf::Tile<T, device>& tile) {
```

Comment:
@rasolca 
Do you have something against embedding this function into `tile.h`?

Since `Buffer` and `Message` concepts are separated, it wouldn't add any dependency to MPI.

Instead, having it in a separate file like this forces users to remember to include it every time they need to send a `Tile`, which IMHO is not straightforward.